bin/ColorScript/README.md

```diff
@@ -0,0 +1,3 @@
+# ColorScriptExecutable
+
+A description of this package.
```

Comment:
Can we add the usage instructions and maybe some details about how to update and re-build this script in here?